helm/k8s-dns-node-cache-app/templates/configmap.yaml

```diff
@@ -50,6 +50,6 @@ data:
         reload
         loop
         bind {{ .Values.cluster.kubernetes.DNS.IP }}
-        forward . __PILLAR__UPSTREAM__SERVERS__
+        forward . __PILLAR__CLUSTER__DNS__
```

Comment:
Did you test this? I don't think this will work.
This config comes straight from the documentation and is meant to be like that.
I might be missing something though.

Please add a changelog entry

Reply:
Was tested by adidas and deployed in deu01 without issue for 2 weeks almost
 https://gigantic.slack.com/archives/C6L8J93N0/p1663935100701069?thread_ts=1663932339.907049&cid=C6L8J93N0

![Screenshot 2022-10-05 at 15 13 17](https://user-images.githubusercontent.com/1133954/194068887-5f43b3f4-5eb9-4816-af22-f766f7955233.png)

Reply:
Added changelog